packages/shylu/shylu_dd/frosch/test/Thyra_Xpetra_Elasticity/ParameterLists/ParameterList_TwoLevelPreconditioner_GDSW_ML.xml

```diff
@@ -0,0 +1,138 @@
+<ParameterList name="Thyra Example">
+    <Parameter name="Linear Solver Type" type="string" value="Belos"/>
+    <ParameterList name="Linear Solver Types">
+        <ParameterList name="Belos">
+            <Parameter name="Solver Type" type="string" value="Block GMRES"/>
+            <ParameterList name="Solver Types">
+                <ParameterList name="Block GMRES">
+                    <Parameter name="PreconditionerPosition" type="string" value="left"/>
+                    <Parameter name="Block Size" type="int" value="1"/>
+                    <Parameter name="Convergence Tolerance" type="double" value="1e-4"/>
+                    <Parameter name="Maximum Iterations" type="int" value="100"/>
+                    <Parameter name="Output Frequency" type="int" value="1"/>
+                    <Parameter name="Show Maximum Residual Norm Only" type="bool" value="1"/>
+                </ParameterList>
+            </ParameterList>
+        </ParameterList>
+    </ParameterList>
+    <Parameter name="Preconditioner Type" type="string" value="FROSch"/>
+    <ParameterList name="Preconditioner Types">
+        <ParameterList name="FROSch">
+            <Parameter name="FROSch Preconditioner Type"                        type="string"   value="TwoLevelPreconditioner"/>
+            <Parameter name="Dimension"                                         type="int"      value="2"/>
+            <Parameter name="DofsPerNode1"                                      type="int"      value="2"/>
+            <Parameter name="DofsPerNode2"                                      type="int"      value="1"/>
+            <Parameter name="DofOrdering1"                                      type="string"   value="NodeWise"/>
+            <Parameter name="DofOrdering2"                                      type="string"   value="NodeWise"/>
+            <Parameter name="Overlap"                                           type="int"      value="1"/>
+            <Parameter name="TwoLevel"                                          type="bool"     value="true"/>
+            <Parameter name="OverlappingOperator Type"                          type="string"   value="AlgebraicOverlappingOperator"/>
+            <Parameter name="CoarseOperator Type"                               type="string"   value="IPOUHarmonicCoarseOperator"/>
```

Comment:
Could you please also use consistent names for the parameter list files to indicate that you are using IPOUHarmonicCoarseOperator?

Reply:
See previous comment.